### ISSUE TITLE
Updated to 33.1.1.406 - Can't Compile

### ISSUE THREAD

**Issue by Bilbosax** [Gamua/Adobe-Runtime-Support#736]
I just signed up for the first paid version of AIR and broke everything ;) LOL

So I installed a fresh version of Flex 4.16.1, downloaded the new version of AIR 33.1.1.406 and overlayed it on the Flex SDK, and then placed my new adt.lic file in the lib folder.  I then pointed intelliJ to my new SDK folder, compiled, and I get this error:

"WARNING: could not locate iTunesMetadata.plist in archive!
WARNING: could not locate Payload/MyApp.app/SC_Info/MyApp.sinf in archive!
ERROR: Install failed. Got error "ApplicationVerificationFailed" with code 0xe8008001: Failed to verify code signature of /private/var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.sP0ZYd/extracted/Payload/MyApp.app : 0xe8008001 (An unknown error has occurred.)"

The app still compiles fine with the older 33.1.1.300 version.  I haven't changed any provisioning profiles or certificates.  Can anyone give me an idea of why this is occurring and what I can do to fix it?

**Comment by ajwfrost** [comment]
Duplicate of #511 and a few other issues ... please try using the `adt.jar` from the below link, in place of the one in the SDK's `lib` folder:
https://transfer.harman.com/message/4SPAINrOXZzGEDmaAEUqNF

**Comment by Bilbosax** [comment]
Well, this certainly helped with the code signing, but it goofed up the display in my app.  It squished my app cutting of the top and bottom of the screen, which threw off the alignment of the buttons/text.  I've attached a screenshot. Any thoughts?


![C096258B-F59F-4F2A-A01F-D114653F1DBB](https://user-images.githubusercontent.com/23325601/111800239-23b3d680-88a2-11eb-9149-527362c9c763.png)

**Comment by FliplineStudios** [comment]
Have you created your own LaunchScreen storyboard (not launch images) to handle screen sizing on iOS devices?  I believe the free tier of AIR uses a built-in storyboard for the splash screen that would handle this on its own, but once that splash screen is gone for the paid version I think you need to supply your own storyboard and add it to the application XML and packaging.

**Comment by Bilbosax** [comment]
Flipline, thanks for the suggestion.  The last update I made to the app store was before a launchscreen storyboard was necessary, so no, I don't have one and I will get one made.

But I have a bigger problem.  I had overlaid so many SDK's over the years that I decided to install a fresh new Flex 4.16.1 SDK, a fresh AIR 33.1.1.406 SDK overlay, installed this new adt.jar this morning from the link above, and place my adt.lic next to it.  This afternoon, it does not work, I am getting the error:

"WARNING: could not locate iTunesMetadata.plist in archive!
WARNING: could not locate Payload/MyApp.app/SC_Info/MyApp.sinf in archive!
ERROR: Install failed. Got error "ApplicationVerificationFailed" with code 0xe8008015: Failed to verify code signature of /private/var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.sOSiNI/extracted/Payload/MyApp.app : 0xe8008015 (A valid provisioning profile for this executable was not found.)"

Has something changed to the adt.jar file you linked to earlier?  I was able to get it to work this morning, but with this fresh install of all the SDK's and adding this new adt.jar, it is not working this afternoon.
